azext_iot/tests/digitaltwins/test_dt_data_history_lifecycle_int.py

```diff
@@ -76,11 +83,15 @@ def test_dt_data_history_adx(self):
         table_name = f"tb_{generate_generic_id()}"
         consumer_group = f"cg-{generate_generic_id()}"
         self.add_eventhub_consumer_group(consumer_group=consumer_group)
+        user_identity = self.ensure_user_identity()["id"]
+        # TODO: lower sleep time to necessary amount
+        sleep(50)
```

Comment:
Is this still a `TODO`? Is there anything we could loop / poll for?

Reply:
This is a copy paste for having the User Assigned Identity be fully created (and associated in the graph)... Sadly I have no idea how to check if the UAI has been fully created aside from trying to use it in another command (and have the command fail). 50 seconds is the sweet spot I hit for previous UAI tests

None of the fields suggest any progress: 
![image](https://user-images.githubusercontent.com/73560279/209020425-2a867c64-583f-4fb9-9902-a4ea8211226e.png)


I kept the TODO so it is easier to find if I think of a better way for the UAI tests